lib/Runtime/Library/JavascriptNumber.cpp

```diff
@@ -230,6 +230,14 @@ namespace Js
 
     static const double d1_0 = 1.0;
 
+#ifdef __clang__
+    // todo : make this fast!
+    // Clang doesn't support so many things under the fast version below
```

Comment:
I think this is only called by the JIT.  And this change will break that.
Might want to put it under #ifdef ENABLE_NATIVE_CODEGEN instead?  So we will hit this and fix it property when we work on x86 JIT?